### PR TITLE
Feat/ux new inputs

### PR DESCRIPTION
**Pull request by AlexNti**
This PR introduces the new UI/UX designs of the input elements.

https://app.zenhub.com/workspaces/product-5ea1534e9ff5df7daace00d5/issues/netdata/product/2656

### CODE REVIEW

File: src/components/input/input.js [hunk 1]
```diff
@@ -55,6 +56,7 @@ export const TextInput = ({
   isDirty,
   value,
   inputRef,
+  size = "large ",
```
Comment (jacekkolasa) [inline src/components/input/input.js:59]: is it a typoe?
```suggestion
  size = "large",
```

**Review by jacekkolasa — APPROVED** [review]
good stuff! Left some minor (optional) comments..

File: src/components/input/use-input-styles.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+}) => ({
+  success: success,
+  error: error,
+  disabled: disabled,
+  default: defaultColor,
+})
+const useInputStyles = ({ size = "large", disabled, success, error, focused }) => {
+  const status = success ? "success" : error ? "error" : disabled ? "disabled" : "default"
+
+  const sizeStyles = React.useCallback(() => {
```
[the remark this answers is not in the document]
Comment (AlexNti) [reply src/components/input/use-input-styles.js:17]: I agree with you Jacek, it makes more sense to replace it with useMemo.

File: src/components/input/use-input-styles.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        }
+      case "small":
+        return {
+          height: "34px",
+          padding: [1.8, 3, 1.8, 3],
+        }
+      default: {
+        return {
+          height: "42px",
+          padding: [2.8, 4, 2.8, 4],
```
[the remark this answers is not in the document]
Comment (AlexNti) [reply src/components/input/use-input-styles.js:32]: Oups , I might have misread the padding function. I thought that it only works with 4 arguments. Should i change it to 2?
 
For the numbers i was trying to be as close as possilbe to the desings padding, thats why i used not integer values. 
Shall i switch those numbers to integer?

File: src/components/input/input.js (hunk 1 above)
Comment (AlexNti) [reply src/components/input/input.js:59]: Yes, this is  a typo

File: src/components/input/use-input-styles.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+          size: "1px",
+          type: "solid",
+          color: makeColor({ defaultColor: "inputBorderHover" })[status],
+          side: "all",
+        },
+      },
+    }
+  }, [status, sizeStyles, focused])
+
+  const iconContainer = React.useCallback(
```
[the remark this answers is not in the document]
Comment (AlexNti) [reply src/components/input/use-input-styles.js:63]: I will change to be alligned with the rest of the code.

**Review by burbuli8ra — APPROVED** [review]

**Pull request merged**